include/cutlass/half.h

```diff
@@ -224,7 +224,7 @@ struct alignas(2) half_t {
     #endif
 
     uint16_t sign = uint16_t((s >> 16) & 0x8000);
-    int16_t exp = uint16_t(((s >> 23) & 0xff) - 127);
+    int16_t exp = int16_t(((s >> 23) & 0xff) - 127);
```

Comment:
The "Don't Repeat Yourself" principle suggests using `auto` here.

```suggestion
    auto exp = int16_t(((s >> 23) & 0xff) - 127);
```